src/state/types.ts

```diff
@@ -37,6 +37,8 @@ export interface GameState {
     boughtPlant: boolean;
     boughtHabitat: boolean;
     uidsUsedForAbilityRefresh: AnimalUID[];
+    currentAbilityCard?: PlantCard | AnimalCard;
```

Comment:
Yes, but this is tracking the ability card you used to enable the states on the drop button, also to know which card was already used

Reply:
I kept an additional property for now to keep track of the card that was selected for abilities, because when we actually assign the current ability we need the name already, so an intermediate step was needed